libkatana_python_native/src/PropertyGraph.cpp

```diff
@@ -1030,11 +1035,11 @@ DefPropertyGraph(py::module& m) {
 
   cls.def(
       "write",
-      [](PropertyGraph& self, const std::string* path,
+      [](PropertyGraph& self, const katana::URI& path,
```

Comment:
This is not gonna work since `katana::URI` isn't wrapped for python. Ping me tomorrow and I'll have a better way to handle this.

Reply:
pxd and pybind11 are totally separate. Currently some things appear in both places. However I think in this case we don't actually need any of the Cython side changes since all that code is legacy and I think can actually be removed (I'll make a PR to do that).